plugin/completion/lib_complete.py

```diff
@@ -213,16 +213,16 @@ def init(self, view, settings):
             self.timer = Timer(Completer.timer_period, self.__remove_old_TUs)
             self.timer.start()
 
-    def complete(self, view, cursor_pos, show_errors):
+    def complete(self, view, cursor_pos, current_job_id):
         """ This function is called asynchronously to create a list of
         autocompletions. Using the current translation unit it queries libclang
         for the possible completions. It also shows compile errors if needed.
 
         Args:
             view (sublime.View): current view
             cursor_pos (int): sublime provided poistion of the cursor
-            show_errors (bool): controls if we need to show errors
```

Comment:
Nah, removed this thing completely :) The documentation for arguments only in base method.